chain/src/chain.rs

```diff
@@ -176,6 +176,7 @@ impl<CS: ChainStore + 'static> ChainService<CS> {
         ChainService { shared, notify }
     }
 
+    #[allow(clippy::zero_ptr, clippy::drop_copy)]
```

Comment:
Please add a comment why exclude these linter rules.

Reply:
I think we should add a wrapper method to create the crossbeam channel temporarily, and add flags to that wrapper method.

Reply:
> I think we should add a wrapper method to create the crossbeam channel temporarily, and add flags to that wrapper method.

I think this behavior has no sense other than adding complexity. `select!` is a Macro with uncertain parameters, which means that if you want to wrap it, you also need a Macro with uncertain parameters, macro set macros, adding complexity, and almost no income